gCoffee/app/src/main/java/com/mborowiec/gcoffee/Order.java

```diff
@@ -108,21 +109,36 @@ private void makeOrder(String coffee) {
     }
 
     /**
-     * Checks whether the menu contains the ordered item. If it does, asks to confirm order.
+     * Checks whether the menu contains the ordered item.
+     * @param item  item ordered by user
+     * @return boolean value
+     */
+    public boolean checkMenu(String item, String[] coffeeList) {
+        if (item != null) item = item.toLowerCase();
+        for (int i = 0; i < coffeeList.length; i++) {
```

Comment:
It's not necessarily better, but did you consider to use streams for this operation? It might at least make sense to try to write it once as a learning opportunity...

Reply:
I actually haven't thought about it, I will try and implement it this way